### PR TITLE
[TDV-64] Feature/unify names

### PR DESCRIPTION
**Pull request by Electr1Xx**

### CODE REVIEW

File: src/components/User/Assets/AssetRequests/AssetRequests.Index.vue [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import AssetRequestList from './components/AssetRequestList'
+
+export default {
+  components: {
+    AssetRequestList
+  }
+}
+</script>
+
+<style scoped>
```
Comment (letehaha) [inline src/components/User/Assets/AssetRequests/AssetRequests.Index.vue]: Remove `scoped` attr pls

File: src/components/User/Assets/AssetRequests/AssetRequests.Show.vue [hunk 2]
```diff
@@ -13,123 +13,123 @@
...
     <div
-      class="token-requests-show"
-      v-else-if="tokenRequest.id && !isRejecting"
+      class="asset-requests-show"
+      v-else-if="assetRequest.id && !isRejecting"
     >
       <h2>
-        <template v-if="tokenRequest.type === 'create_asset'">
-          Token creation request
+        <template v-if="assetRequest.type === 'create_asset'">
```
Comment (letehaha) [inline src/components/User/Assets/AssetRequests/AssetRequests.Show.vue]: Let's look at `js-sdk` and try to find `assetRequest` types as constant. If this information does not exist, please, move these values (`create_asset` and `update_asset`) to some constant.

File: src/components/User/Assets/AssetRequests/AssetRequests.Show.vue [hunk 3]
```diff
@@ -216,20 +216,20 @@
 import { Sdk } from '@/sdk'
 import { ASSET_POLICIES_VERBOSE } from '@/constants'
 import { confirmAction } from '../../../../js/modals/confirmation_message'
```
Comment (letehaha) [inline src/components/User/Assets/AssetRequests/AssetRequests.Show.vue]: Please alias

File: src/components/User/Assets/AssetRequests/AssetRequests.Show.vue [hunk 4]
```diff
@@ -266,9 +266,9 @@ export default {
   computed: {
     isCancellable () {
       const isPending =
-        this.tokenRequest.state === 'pending'
+        this.assetRequest.state === 'pending'
```
Comment (letehaha) [inline src/components/User/Assets/AssetRequests/AssetRequests.Show.vue]: Create some constant for `pending` state. And replace this string to the constant value through component

File: src/components/User/Assets/AssetRequests/AssetRequests.Show.vue [hunk 5]
```diff
@@ -17,11 +17,11 @@
       v-else-if="assetRequest.id && !isRejecting"
     >
       <h2>
-        <template v-if="assetRequest.type === 'create_asset'">
+        <template v-if="assetRequest.type === ASSET_REQUEST_TYPES[createAsset]">
```
Comment (letehaha) [inline src/components/User/Assets/AssetRequests/AssetRequests.Show.vue]: `ASSET_REQUEST_TYPES.createAsset`

File: src/components/User/Assets/AssetRequests/AssetRequests.Show.vue [hunk 6]
```diff
@@ -172,7 +172,7 @@
         </div>
       </template>
 
-      <template v-if="assetRequest.state === 'rejected'">
+      <template v-if="assetRequest.state === CREATE_ASSET_REQUEST_STATES['rejected'].codeVerbose">
```
Comment (letehaha) [inline src/components/User/Assets/AssetRequests/AssetRequests.Show.vue]: `CREATE_ASSET_REQUEST_STATES.rejected.codeVerbose`

File: src/components/User/Assets/AssetRequests/AssetRequests.Show.vue [hunk 7]
```diff
@@ -160,7 +160,7 @@
         />
       </div>
 
-      <template v-if="assetRequest.state !== 'pending'">
+      <template v-if="assetRequest.state !== CREATE_ASSET_REQUEST_STATES['pending'].codeVerbose">
```
Comment (letehaha) [inline src/components/User/Assets/AssetRequests/AssetRequests.Show.vue]: `CREATE_ASSET_REQUEST_STATES.pending.codeVerbose`

File: src/components/User/Assets/AssetRequests/AssetRequests.Show.vue [hunk 8]
```diff
@@ -184,7 +184,7 @@
 
       <div
         class="asset-requests-show__buttons"
-        v-if="assetRequest.state === 'pending'"
+        v-if="assetRequest.state === CREATE_ASSET_REQUEST_STATES['pending'].codeVerbose"
```
Comment (letehaha) [inline src/components/User/Assets/AssetRequests/AssetRequests.Show.vue]: `CREATE_ASSET_REQUEST_STATES.pending.codeVerbose`

File: src/components/User/Assets/AssetRequests/AssetRequests.Show.vue (hunk 5 above)
Comment (Electr1Xx) [reply src/components/User/Assets/AssetRequests/AssetRequests.Show.vue]: https://github.com/tokend/admin-panel/pull/42/commits/c413f75a681116f1ac68e3a7c11c57ae2c8734ef

**Pull request merged**